config.js

```diff
@@ -907,6 +909,8 @@ config.ENDPOINT_SSL_PORT = Number(process.env.ENDPOINT_SSL_PORT) || 6443;
 config.ENDPOINT_SSL_STS_PORT = Number(process.env.ENDPOINT_SSL_STS_PORT) || (process.env.NC_NSFS_NO_DB_ENV === 'true' ? -1 : 7443);
 config.ENDPOINT_SSL_IAM_PORT = Number(process.env.ENDPOINT_SSL_IAM_PORT) || -1;
 config.ALLOW_HTTP = false;
+config.ALLOW_HTTPS_METRICS = false;
```

Comment:
why false by default?

Reply:
@romayalon ODF enabling https dosn't make sense without the `Service` changes. But we can enable it by default and can be used with service yaml changes and non-containerized can use http port without any config changes.

Reply:
we wanted to support it on ODF and on NC, so I don't understand why it's false
which https changes are you referring to?

Reply:
yes got your point, Will update value with `true` and enable HTTPS. ODF also can fetch metrics from SSL port without any config change in Noobaa side.